composer.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "blesta/digitaloceanmodule",
+    "description": "DigitalOcean",
+    "license": "proprietary",
```

Comment:
I just had that from a while ago sorry that was from the form post I will dump that and put MIT on it then.